scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Record.java

```diff
@@ -521,14 +526,14 @@ private Record(ProtocolVersion version, int epoch, long sequenceNumber, long rec
 		plaintext.writeBytes(getBlockCipherMac(writeState, compressedFragment));
 
 		// determine padding length
-		int ciphertextLength = compressedFragment.length + writeState.getCipherSuite().getMacLength() + 1;
-		int smallestMultipleOfBlocksize = writeState.getRecordIvLength();
-		while ( smallestMultipleOfBlocksize <= ciphertextLength) {
-			smallestMultipleOfBlocksize += writeState.getRecordIvLength();
+		int ciphertextLength = compressedFragment.length + writeState.getMacLength() + 1;
+		int blocksize = writeState.getRecordIvLength();
+		int paddingLength = ciphertextLength % blocksize;
+		if (0 < paddingLength) {
```

Comment:
To be frank: nobody cares for years. So I don't think, this detail is worth to do much more :-).

Reply:
My feeling, I blocked when reading this part of the code just because of the variable naming.
When I faced that, generally I try to fix it.
(I often face that when I read my code long time after. ^^)

I you want I can do a PR just with this little change of code and the changes about info/warning log ?

Reply:
OK,
May be remainingBytes := lastBlocksBytes but you may go for what ever you want :-).

Reply:
OK pour lastBlocksBytes ! 
I wait you merge this PR, then create a new one.

Reply:
OK, so LGTM?